libs/fades.liq

```diff
@@ -115,7 +124,7 @@ def fade.out(~id="fade.out",~duration=3.,
   delay = fun () -> !duration
   s = on_end(delay=delay,start_fade,s)
   s = on_track(stop_fade,s)
-  amplify(id=id,override="",apply,s)
+  fade.scale(id=id,apply,s)
```

Comment:
Isn't that gonna force all sources to be `video=1+#a`?

Reply:
Nope, see https://github.com/savonet/liquidsoap/pull/1135/files#diff-05a71795aba4f59cf7098a4c39224665R48
Video effects don't actually require to have video (which I think is ok for this reason).

Reply:
Cool. If you've tested it then it's 👍 for me!